mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -56,8 +56,8 @@ class BackgroundTasksManager : BroadcastReceiver() {
                     return
                 }
                 val bundle = intent.getBundleExtra(TaskerIntent.EXTRA_BUNDLE) ?: return
-                val itemName = bundle.getString(ItemPickerActivity.EXTRA_ITEM_NAME)
-                val state = bundle.getString(ItemPickerActivity.EXTRA_ITEM_STATE)
+                val itemName = bundle.getString(AbstractItemPickerActivity.EXTRA_ITEM_NAME)
+                val state = bundle.getString(AbstractItemPickerActivity.EXTRA_ITEM_STATE)
```

Comment:
Since the bundle name references `TaskerIntent`, I think it's better to use `TaskerItemPickerActivity` here.

Reply:
IMO the child classes of `AbstractItemPickerActivity` should always send back a bundle containing item name and state, so we always need these two constants.